test/Models/Catalog/CatalogItems/GeoJsonCatalogItemSpec.ts

```diff
@@ -73,12 +73,12 @@ describe("GeoJsonCatalogItemSpec", () => {
                 type: "Polygon",
                 coordinates: [
                   [
-                    [144.80667114257812, -32.96258644191746],
+                    [144.806671142578125, -32.96258644191746],
```

Comment:
You've already done this now, but I fixed this problem in #7149 back in May (with an identical fix), so I think you could have saved yourselves a bit of effort by first merging the fixes you had laying around already.

Reply:
Thanks Peter, I should have done a scan for existing lint PRs first. I've merged your changes in :)